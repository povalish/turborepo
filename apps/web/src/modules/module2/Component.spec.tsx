import { render } from '@testing-library/react';
import { Component } from './Component';

describe('test component', () => {
  it('renders component unchanged', () => {
    const { container } = render(<Component />);
    expect(container).toMatchSnapshot();
  });
});
