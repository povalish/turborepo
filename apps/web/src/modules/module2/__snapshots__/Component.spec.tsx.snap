// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component renders component unchanged 1`] = `
<div>
  <div>
    Component
  </div>
</div>
`;
